django_ipam/templates/admin/django-ipam/change_form.html

```diff
@@ -0,0 +1,52 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls %}
+{% load static %}
+
+{% block after_related_objects %}
+    <li style="list-style-type:none">
+      <div id="graph">
+      </div>
+      <div style="padding:10px">
+        <ul style="text-align:justify;">
+          {% for i in total %}
+            {% if i in used_ip %}
+              <li style="display:inline;text-align:center;">
+                <a style='color:red;'>
+                  {{i}}
+                </a>
+              </li>
+            {% else %}
+              <li style="display:inline;text-align:center;">
+                <a href="/admin/django_ipam/ipaddress/add/?_to_field=id&_popup=1&ip_address={{i}}" onclick='return showAddAnotherPopup(this);'>
```

Comment:
see github highlight those `&` in red? Try rewriting `&` as `&amp;` (correct HTML entity code for ampersands) and check if the link still works, should work